src/styles/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+
+  &__logo {
+    object-fit: none;
+    width: 40px;
+    margin: 10px 0;
+    margin-left: 50px;
+  }
+
+  &__list {
+    all: unset;
+    display: flex;
+    height: 60px;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    list-style: none;
+    line-height: 13px;
+    margin-right: 44px;
+  }
+
+  &__link {
+    all: unset;
+    display: inline-block;
+    padding: 25px 0;
+
+    &--active {
+      color: #00acdc;
+      position: relative;
+    }
+
+    &--active::after {
+      content: "";
+      position: absolute;
+      top: 90%;
+      left: 0%;
+      width: 39px;
```

Comment:
Please try to use width: 100% so that this blue stripe is full width of nav item